File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Orca/ORCALibGetClaimDiagnosisInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Orca
{
    [Serializable]
    public class ORCALibGetClaimDiagnosisInfo
    {
        private String _updDate;
        private String _pkoif5101;
        private String _license;
        private String _gwa;
        private String _gwaName;
        private String _categoriTd;
        private String _categoriName;
        private String _code;
        private String _system;
        private String _codeName;
        private String _startDate;
        private String _endDate;
        private String _outCome;
        private String _docId;

        public String UpdDate
        {
            get { return this._updDate; }
            set { this._updDate = value; }
        }

        public String Pkoif5101
        {
            get { return this._pkoif5101; }
            set { this._pkoif5101 = value; }
        }

        public String License
        {
            get { return this._license; }
            set { this._license = value; }
        }

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String GwaName
        {
            get { return this._gwaName; }
            set { this._gwaName = value; }
        }

        public String CategoriTd
        {
            get { return this._categoriTd; }
            set { this._categoriTd = value; }
        }

        public String CategoriName
        {
            get { return this._categoriName; }
            set { this._categoriName = value; }
        }

        public String Code
        {
            get { return this._code; }
            set { this._code = value; }
        }

        public String System
        {
            get { return this._system; }
            set { this._system = value; }
        }

        public String CodeName
        {
            get { return this._codeName; }
            set { this._codeName = value; }
        }

        public String StartDate
        {
            get { return this._startDate; }
            set { this._startDate = value; }
        }

        public String EndDate
        {
            get { return this._endDate; }
            set { this._endDate = value; }
        }

        public String OutCome
        {
            get { return this._outCome; }
            set { this._outCome = value; }
        }

        public String DocId
        {
            get { return this._docId; }
            set { this._docId = value; }
        }

        public ORCALibGetClaimDiagnosisInfo() { }

        public ORCALibGetClaimDiagnosisInfo(String updDate, String pkoif5101, String license, String gwa, String gwaName, String categoriTd, String categoriName, String code, String system, String codeName, String startDate, String endDate, String outCome, String docId)
        {
            this._updDate = updDate;
            this._pkoif5101 = pkoif5101;
            this._license = license;
            this._gwa = gwa;
            this._gwaName = gwaName;
            this._categoriTd = categoriTd;
            this._categoriName = categoriName;
            this._code = code;
            this._system = system;
            this._codeName = codeName;
            this._startDate = startDate;
            this._endDate = endDate;
            this._outCome = outCome;
            this._docId = docId;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/orca/ORCALibGetClaimDiagnosisInfo.java
package nta.med.data.model.ihis.orca;

public class ORCALibGetClaimDiagnosisInfo {
	private String updDate           ;
	private Double pkoif5101          ;
	private String license            ;
	private String gwa                ;
	private String gwaName           ;
	private String categoriTd        ;
	private String categoriName      ;
	private String code               ;
	private String system             ;
	private String codeName          ;
	private String startDate         ;
	private String endDate           ;
	private String outCome           ;
	private String docId             ;
	public ORCALibGetClaimDiagnosisInfo(String updDate, Double pkoif5101,
			String license, String gwa, String gwaName, String categoriTd,
			String categoriName, String code, String system, String codeName,
			String startDate, String endDate, String outCome, String docId) {
		super();
		this.updDate = updDate;
		this.pkoif5101 = pkoif5101;
		this.license = license;
		this.gwa = gwa;
		this.gwaName = gwaName;
		this.categoriTd = categoriTd;
		this.categoriName = categoriName;
		this.code = code;
		this.system = system;
		this.codeName = codeName;
		this.startDate = startDate;
		this.endDate = endDate;
		this.outCome = outCome;
		this.docId = docId;
	}
	public String getUpdDate() {
		return updDate;
	}
	public void setUpdDate(String updDate) {
		this.updDate = updDate;
	}
	public Double getPkoif5101() {
		return pkoif5101;
	}
	public void setPkoif5101(Double pkoif5101) {
		this.pkoif5101 = pkoif5101;
	}
	public String getLicense() {
		return license;
	}
	public void setLicense(String license) {
		this.license = license;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getCategoriTd() {
		return categoriTd;
	}
	public void setCategoriTd(String categoriTd) {
		this.categoriTd = categoriTd;
	}
	public String getCategoriName() {
		return categoriName;
	}
	public void setCategoriName(String categoriName) {
		this.categoriName = categoriName;
	}
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
	public String getSystem() {
		return system;
	}
	public void setSystem(String system) {
		this.system = system;
	}
	public String getCodeName() {
		return codeName;
	}
	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}
	public String getStartDate() {
		return startDate;
	}
	public void setStartDate(String startDate) {
		this.startDate = startDate;
	}
	public String getEndDate() {
		return endDate;
	}
	public void setEndDate(String endDate) {
		this.endDate = endDate;
	}
	public String getOutCome() {
		return outCome;
	}
	public void setOutCome(String outCome) {
		this.outCome = outCome;
	}
	public String getDocId() {
		return docId;
	}
	public void setDocId(String docId) {
		this.docId = docId;
	}
	
}
